package.json

```diff
@@ -139,6 +139,9 @@
 		"rimraf": "^3.0.2",
 		"typescript": "^4.6.3"
 	},
+	"devDependencies": {
+		"@types/busboy": "^1.5.0"
```

Comment:
How did this end up in root package?

Reply:
I ran `npm i --save-dev @types/busboy` after getting an error.

Reply:
Try running this `npm remove @types/busboy && git pull origin main --rebase`  OR `npm remove @types/busboy && npm i --save-dev -w wrangler @types/busboy`